app/controllers/cards_controller.rb

```diff
@@ -0,0 +1,7 @@
+require 'Date'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.